apps/cowswap-frontend/src/common/containers/PendingOrderNotification/index.cosmos.tsx

```diff
@@ -0,0 +1,34 @@
+import { USDC_MAINNET, WBTC } from '@cowprotocol/common-const'
+import { OrderKind } from '@cowprotocol/cow-sdk'
+import { CurrencyAmount } from '@uniswap/sdk-core'
+
+import styled from 'styled-components/macro'
+
+import { UiOrderType } from 'utils/orderUtils/getUiOrderType'
+
+import { PendingOrderNotification } from './index'
+
+const defaultProps = {
```

Comment:
great you did the cosmos!

Is it supposed to look like this? 
<img width="1467" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/f1191bc0-80d8-49a9-a2bd-18b563dad7a1">